service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/CatalogHandlerUtils.java

```diff
@@ -607,8 +829,15 @@ protected ViewMetadata commit(ViewOperations ops, UpdateTableRequest request) {
     return ops.current();
   }
 
-  private int maxCommitRetries() {
+  @VisibleForTesting
+  public int maxCommitRetries() {
     return configurationStore.getConfiguration(
         realmContext, FeatureConfiguration.ICEBERG_COMMIT_MAX_RETRIES);
   }
+
+  @VisibleForTesting
+  public boolean isRollbackCompactionEnabled() {
```

Comment:
could be package-private

Reply:
Sorry, I missed that. However, would it not be nicer to make this flag a constructor parameter rather than mocking test values into it?